src/TransportCompatibilityTests/RabbitMQV1/app.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="NServiceBus.Core" publicKeyToken="9fc386479f8a226c" culture="neutral" />
```

Comment:
@ramonsmits they were auto-generated by NuGet. I think they will probably go away after all the packages are upgraded.

If you want, I can try and remove them now and see if things still work.